scripts/updateNpm.js

```diff
@@ -0,0 +1,47 @@
+// @ts-check
+
+import { writeFileSync } from 'node:fs';
+import { promisify } from 'node:util';
+import { exec } from 'node:child_process';
+import { componentsSchema } from '../src/lib/schemas.js';
+import components from '../src/routes/components/components.json' assert { type: 'json' };
+
+const execAsync = promisify(exec);
+
+const data = componentsSchema.parse(components);
+
+const { versions, dates, support } = await Promise.all(
+	data.map(async (pkg) => {
+		try {
+			return await processPackage(pkg);
+		} catch (error) {
+			console.log(error.message);
+		}
+	})
```

Comment:
Unnecessary `async`/`await`
```suggestion
	data.map((pkg) => processPackage(pkg)
		.catch (error => console.log(error.message))
	})
```